vms/platformvm/state/state_helpers.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/vms/platformvm/signer"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
+)
+
+func getStakerColdAttributes(chain Chain, stakerID ids.ID) (*StakerColdAttributes, error) {
```

Comment:
these methods are just to avoid code duplication across `state.State `and `state.Diff`.
MerkleDB implementation is likely to have different implementation and not require these methods